app/tests/src/cardano-wallet.js

```diff
@@ -184,15 +184,21 @@ describe('successful transaction fee computation', () => {
     mockNet.mockBulkAddressSummaryEndpoint()
     mockNet.mockUtxoEndpoint()
 
-    assert.equal((await wallets.used.getTxPlan(myAddress, 47, 0)).fee, 179288)
+    assert.equal(
+      (await wallets.used.getTxPlan({address: myAddress, coins: 47, donationAmount: 0})).fee,
+      179288
+    )
     mockNet.clean()
   })
 
   it('should compute the right transaction fee for shorter outgoing address', async () => {
     const mockNet = mockNetwork(mockConfig2)
     mockNet.mockUtxoEndpoint()
 
-    assert.equal((await wallets.used.getTxPlan(shortAddress, 47, 0)).fee, 177838)
+    assert.equal(
+      (await wallets.used.getTxPlan({address: shortAddress, coins: 47, donationAmount: 0})).fee,
```

Comment:
@elfinka could you describe in the PR description what issue did you find in the tests and how do your changes address the problem(s)?

Reply:
Added description